metadata/lang/es.yml

```
@@ -31,18 +31,18 @@ figure: "Figura"
 # endmatter
 key-points: "Puntos clave"
 faqs: "Preguntas frecuentes"
-#Have questions about this tutorial? Check out the FAQ page for the Introduction to Galaxy Analyses topic to see if your question is listed there. If not, please ask your question on the GTN Gitter Channel or the Galaxy Help Forum: "¿Tienes preguntas sobre este tutorial? Consulta la página de preguntas frecuentes para ver si tu pregunta aparece allí. De lo contrario, haz tu pregunta en GTN Gitter Channel o en el Foro de ayuda de Galaxy"
+#Have questions about this tutorial? Check out the FAQ page for the Introduction to Galaxy Analyses topic to see if your question is listed there. If not, please ask your question on the GTN Gitter Channel or the Galaxy Help Forum: "¿Tienes preguntas sobre este tutorial? Consulta la página de preguntas frecuentes para ver si tu pregunta aparece allí. De lo contrario, haz tu pregunta en en el canal GTN de Gitter o en el Foro de ayuda de Galaxy"
 references: "Referencias"
 feedback: "Retroalimentación"
-feedback-text: "¿Usaste este material como instructor?"
+feedback-text: "¿Utilizaste este material como instructor?"
 feedback-link-text: "Cuéntanos tu experiencia"
 
 glossary: "Glosario"
 citing-tutorial: "Cómo citar este tutorial"
-congrats: "¡Felicitaciones! ¡ Completaste con éxito este tutorial!"
+congrats: "¡Felicitaciones! ¡Completaste con éxito este tutorial!"
 
-want-exend-knowledge: "TODO" # want to extend your knowledge?
-recommended-followup: "TODO" # Follow one of our recommended follow-up trainings
+want-exend-knowledge: "¿Quieres expandir tu conocimientos?" # want to extend your knowledge?
+recommended-followup: “Completa alguno o todos los entrenamientos de seguimiento” # Follow one of our recommended follow-up trainings
```

I think this has a slightly different meaning than the original text in English.
Maybe something like this is more accurate?
```suggestion
recommended-followup: “Continúa tu aprendizaje completando uno de nuestros entrenamientos relacionados recomendados” # Follow one of our recommended follow-up trainings
```